src/NServiceBus.TransactionalSession.AcceptanceTests/NServiceBus.TransactionalSession.AcceptanceTests.csproj

```diff
@@ -10,9 +10,9 @@
   <ItemGroup>
     <PackageReference Include="GitHubActionsTestLogger" Version="2.0.1" />
     <PackageReference Include="Microsoft.NET.Test.Sdk" Version="17.3.1" />
-    <PackageReference Include="NServiceBus.AcceptanceTesting" Version="8.0.0-rc.1" />
     <PackageReference Include="NUnit" Version="3.13.3" />
     <PackageReference Include="NUnit3TestAdapter" Version="4.2.1" />
+    <PackageReference Include="NServiceBus.AcceptanceTests.Sources" Version="8.0.0-rc.1" GeneratePathProperty="true" />
```

Comment:
we wouldn't have to use all the "remove" parts further down if we don't rely on the sources packages. I think that would overall be simpler at the cost of more explicitly code the default servers to the specific custom persistence

Reply:
We use this approach in many persistences as well. At this stage, this is almost an established practice